src/main/java/org/spongepowered/common/mixin/core/block/BlockFenceMixin.java

```diff
@@ -37,14 +39,27 @@
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.common.data.ImmutableDataCachingUtil;
 import org.spongepowered.common.data.manipulator.immutable.block.ImmutableSpongeConnectedDirectionData;
+import org.spongepowered.common.util.DirectionalBlockUtils;
 
 import java.util.HashSet;
+import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 
 @Mixin(BlockFence.class)
 public abstract class BlockFenceMixin extends BlockMixin {
 
+    private static final Map<Direction, PropertyBool> DIRECTION_TO_PROPERTY_MAPPING;
```

Comment:
I'd rather not have a static field at all. Also, the field needs to be prefixed according to the naming standards for mixin-added-by-sponge names, i.e. `impl$DIRECTION_TO_PROPERTY_MAPPING`.